test/unit/lib/getMergedPullRequestsSpec.js

```diff
@@ -120,4 +120,18 @@ describe('getMergedPullRequests', function () {
         return expect(getMergedPullRequests(anyRepo))
             .to.become(expectedResults);
     });
+
+    it('should skip with non-matching parenthesis', function () {
+        const gitLogMessages = [
+            'Merge pull request #3 from kadirahq/greenkeeper-update-all'
```

Comment:
I don't know, but I ran it on the Storybook repo and it crashed. Link to the guilty merge is above in the PR description. Regardless of how it happened, the code that this is replacing is brittle to begin with. 😄